eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/model/SubSnpNoHgvs.java

```diff
@@ -140,11 +190,11 @@ public void setContigOrientation(Orientation contigOrientation) {
         this.contigOrientation = contigOrientation;
     }
 
-    public int getContigStart() {
+    public Long getContigStart() {
```

Comment:
All our code is based on the assumption that the contig start will never be null, so it should be `long` instead of `Long`.

Reply:
🎉